pydicom/charset.py

```diff
@@ -93,7 +94,25 @@ def convert_encodings(encodings):
     # Assume that it is already the python encoding
     # (is there a way to check this?)
     except KeyError:
-        pass
+        # check for some common mistakes in encodings
+        patched_encodings = []
+        patched = {}
+        for x in encodings:
+            if x.startswith('ISO-IR') or x.startswith('ISO IR'):
```

Comment:
Just a suggestion, but would regex `^ISO.IR` be better?

Reply:
Yes, I was undecided about that - but actually it will do no harm, so I think I will change it.